packages/contracts/src/dollar/interfaces/IAMO.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.6.11;
```

Comment:
It's better to make it consistent with the code base:
```
pragma solidity 0.8.19;
```

Reply:
Done! I've also added interfaces to the contracts developed.